index.js

```diff
@@ -331,3 +336,161 @@ function setResponse(respContext, processed) {
   respContext.response.errors = processed?.errors;
   respContext.response.extensions = processed?.extensions;
 }
+
+async function InigoFetchGatewayInfo(token) {
+  token = token || process.env.INIGO_SERVICE_TOKEN
```

Comment:
Can be simplified to
``` js
token ||= process.env.INIGO_SERVICE_TOKEN
```

Reply:
yeah, but I find it more confusing. JS allows you to do things using very short, eventually barely readable syntax 😅